src/main/java/edu/unc/lib/boxc/migration/cdm/MigrationTypeReportCommand.java

```diff
@@ -34,13 +36,14 @@ public Integer call() throws Exception {
         try {
             long start = System.nanoTime();
             init();
-            String numWorks = String.valueOf(typeReportService.countWorks());
-            String numFiles = String.valueOf(typeReportService.countFiles());
-            outputLogger.info("Number of Works: {}", numWorks);
-            outputLogger.info("Number of Files: {}", numFiles);
+            outputLogger.info("Number of Works: {}", typeReportService.countWorks());
+            outputLogger.info("Number of Files: {}", typeReportService.countFiles());
             return 0;
+        } catch (NoSuchFileException e) {
+            outputLogger.info("Cannot generate migration types report. Post migration report not found: {}",
+                    e.getMessage());
         } catch (Exception e) {
-            outputLogger.info("Encountered an error while counting new objects", e);
+            outputLogger.info("Encountered an error while counting new objects: {}", e.getMessage());
```

Comment:
Since I just ran into this while debugging another command, could we include the full stacktrace here? We could send it to the migration.log, which would be `log.error("Encountered an error while counting new objects", e);` (you'll probably have to add a `log` object to this class)